src/pages/atomicdex/api/v20/index.mdx

```diff
@@ -214,17 +205,220 @@ The `EvmNode` object includes the following items for a given coin or token:
+| expiry               | string  | Only used if `type` is `keysend`. Optional, defaults to `3600`. Seconds until the payment expires.                                    |
+
+
+### LightningPaymentFilter
+
+| Parameter            | Type    | Description                                                                                                                           |
+|----------------------|---------|---------------------------------------------------------------------------------------------------------------------------------------|
+| payment_type         | object  | A standard `LightningPaymentType` object.                                                                                             |
+| description          | string  | Optional. A note to indicate the purpose of the invoice.                                                                              |
+| status               | string  | Optional. Accepted values: `pending`, `succeeded`, `failed`.                                                                          |
+| from_amount_msat     | integer | Optional. Minimum amount sent in [millisatoshis](https://bitcoindata.science/bitcoin-units-converter.html)                            |
+| to_amount_msat       | integer | Optional. Maximum amount sent in [millisatoshis](https://bitcoindata.science/bitcoin-units-converter.html)                            |
+| from_fee_paid_msat   | integer | Optional. Minimum transaction fee paid in [millisatoshis](https://bitcoindata.science/bitcoin-units-converter.html)                   |
+| to_fee_paid_msat     | integer | Optional. Maximum transaction fee paid in [millisatoshis](https://bitcoindata.science/bitcoin-units-converter.html)                   |
+| from_timestamp       | string  | Optional. Minimum timestamp (in milliseconds) of payment results to return.                                                           |
+| to_timestamp         | string  | Optional. Maximum timestamp (in milliseconds) of payment results to return.                                                           |
+
+
+### LightningPaymentType
+
+| Parameter    | Type    | Description                                                                                                                           |
+|--------------|---------|---------------------------------------------------------------------------------------------------------------------------------------|
+| type         | object  | Accepted values are `Outbound Payment` or `Inbound Payment`.                                                                          |
+| destination  | string  | Only used if `type` is `Outbound Payment`. A pubkey which will receive the payment.                                                   |
```

Comment:
mmm, I need to fix this, I should make destination optional since right now we can't get all outbound payments to different destinations.

Reply:
DevComment added to track this